rotkehlchen/tests/exchanges/test_bitpanda.py

```diff
@@ -92,13 +90,6 @@
 FIAT_WALLETS_RESPONSE = """{"data":[{"type":"fiat_wallet","attributes":{"fiat_id":"3","fiat_symbol":"CHF","balance":"0.00","name":"CHF Wallet","pending_transactions_count":0},"id":"d8a96548-ceed-4848-9b9e-89fe88a64a44"},{"type":"fiat_wallet","attributes":{"fiat_id":"1","fiat_symbol":"EUR","balance":"1500.51","name":"EUR Wallet","pending_transactions_count":0},"id":"7161d200-3f59-4c4f-ab8e-53378317740c"},{"type":"fiat_wallet","attributes":{"fiat_id":"4","fiat_symbol":"GBP","balance":"0.00","name":"GBP Wallet","pending_transactions_count":0},"id":"07dcb81e-74bd-4b43-8812-59edbdc792a8"},{"type":"fiat_wallet","attributes":{"fiat_id":"8","fiat_symbol":"TRY","balance":"155.22","name":"TRY Wallet","pending_transactions_count":0},"id":"e1526b07-ced7-43a4-a692-aa2e07ee71df"},{"type":"fiat_wallet","attributes":{"fiat_id":"2","fiat_symbol":"USD","balance":"0.00","name":"USD Wallet","pending_transactions_count":0},"id":"0e7d9a1b-4639-48c3-93fc-85156a130ee1"}]}"""  # noqa: E501
 
 
-@pytest.mark.parametrize('mocked_current_prices', [{
-    A_BEST: FVal(20),
-    A_ETH: FVal(3000),
-    A_AXS: FVal(40),
-    A_EUR: FVal(0.85),
-    A_TRY: FVal(0.103),
-}])
```

Comment:
why remove this? Doesn't it now query the oracles always?

Reply:
adjusted the fixture to use the main currency instead. my initial approach removed the fixture entirely, which is why yabir commented.